tornado-assembly/src/bin/tornado-test

```diff
@@ -506,21 +509,25 @@ def processStats(out, stats):
                 name = name[:-16]
 
             if (className + "#" + name in __TORNADO_TESTS_WHITE_LIST__):
+                failedTests[className + "#" + name] = "YES"
                 print(Colors.RED + "Test: " + className + "#" + name + " in whiteList." + Colors.RESET)
```

Comment:
also, can we also print the list of non-whitelisted tests that failed? And this could be displaying in RED.

I see that if we have white-listed and non-whitelisted tests, we print them all as RED. For example:
![Screenshot from 2024-10-08 13-32-17](https://github.com/user-attachments/assets/b3b30207-2f5c-4027-8f53-cd1041e533da)


In this case, I deliberately made some tests to fail.